docs/sdk/sdk-by-examples/simple-governance/bridging-it-all.md

```diff
@@ -0,0 +1,256 @@
+# Application - Bridging it all together
+
+## Application structure
+
+Now, that we have built all the pieces we need, it is time to integrate them into the application. Let us exit the `/x` director go back at the root of the SDK directory.
+
+
+```bash
+// At root level of directory
+cd app
+```
+
+We are ready to create our simple governance application!
+
+*Note: You can check the full file (with comments!) [here](link)*
+
+The `app.go` file is the main file that defines your application. In it, you will declare all the modules you need, their keepers, handlers, stores, etc. Let us take a look at each section of this file to see how the application is constructed.
+
+Secondly, we need to define the name of our application.
+
+```go
+const (
+    appName = "SimpleGovApp"
+)
+```
+
+Then, let us define the structure of our application.
+
+```go
+// Extended ABCI application
+type SimpleGovApp struct {
+    *bam.BaseApp
+    cdc *codec.Codec
+
+    // keys to access the substores
+    capKeyMainStore      *sdk.KVStoreKey
+    capKeyAccountStore   *sdk.KVStoreKey
+    capKeyStakingStore   *sdk.KVStoreKey
+    capKeySimpleGovStore *sdk.KVStoreKey
+
+    // keepers
+    feeCollectionKeeper auth.FeeCollectionKeeper
+    bankKeeper          bank.Keeper
+    stakeKeeper         simplestake.Keeper
+    simpleGovKeeper     simpleGov.Keeper
+
+    // Manage getting and setting accounts
+    accountMapper auth.AccountMapper
+}
+```
+
+- Each application builds on top of the `BaseApp` template, hence the pointer.
+- `cdc` is the codec used in our application.
+- Then come the keys to the stores we need in our application. For our simple governance app, we need 3 stores + the main store.
+- Then come the keepers and mappers.
+
+Let us do a quick reminder so that it is  clear why we need these stores and keepers. Our application is primarily based on the `simple_governance` module. However, we have established in section [Keepers for our app](module-keeper.md) that our module needs access to two other modules: the `bank` module and the `stake` module. We also need the `auth` module for basic account functionalities. Finally, we need access to the main multistore to declare the stores of each of the module we use.
+
+## CLI and Rest server 
+
+We will need to add the newly created commands to our application. To do so, go to the `cmd` folder inside your root  directory:
+
+```bash
+// At root level of directory
+cd cmd
+```
+`simplegovd` is the folder that stores the command for running the server daemon, whereas `simplegovcli` defines the commands of your application.
+
+### Application CLI 
+
+**File: [`cmd/simplegovcli/maing.go`](https://github.com/cosmos/cosmos-sdk/blob/fedekunze/module_tutorial/examples/simpleGov/cmd/simplegovcli/main.go)**
```

Comment:
use develop branch?

Reply:
it's not on develop yet, need to ask fede